### ISSUE TITLE
Accuracy

### ISSUE THREAD

**Issue by J3r3myPerera** [dataquestio/project-walkthroughs#11]
Can you give a method to get the accuracy apart from the precision score

**Comment by Kalaweksh** [comment]
If you want more insight into a model's predictions, `sklearn.metrics.classification_report` is a nice method that returns the precision, recall and f1-score as either a nicely formatted string (or a dict, if you would prefer).
